float32 belief_thresh
---
Place[] places
